config/checkstyle/checkstyle.xml

```diff
@@ -39,6 +39,11 @@
           <property name="option" value="top"/>
           <property name="ordered" value="true"/>
         </module>
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="(?i)log(ger)?\.(debug|info|warn|error)\(.*%[sd]"/>

```

Comment:
also probably not important but if the regex can no match on things with String.format that would be good, or maybe an entirely different check is using log. with string.format and we say to just use native log. capabilities with {}